libs/dal/src/repositories/message-template/message-template.schema.ts

```diff
@@ -89,6 +89,13 @@ messageTemplateSchema.index({
   'triggers.identifier': 1,
 });
 
+messageTemplateSchema.index({
+  _environmentId: 1,
+  _layoutId: 1,
+  _feedId: 1,
+  deleted: 1,
```

Comment:
Is `deleted` auto added by `mongoose-delete` on every query?

Reply:
Yes it does